.github/workflows/createAndTestBetaReleaseBranch.yml

```diff
@@ -0,0 +1,81 @@
+name: Create and Test Beta Release Branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create_branch:
+    name: 'Create Branch'
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+      branch: ${{ steps.branch.outputs.branch }}
+      result: ${{ steps.result.outputs.result }}
+    env:
+      RELEASE_TYPE: 'beta'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: 'main'
+          ssh-strict: false
+          token: ${{ secrets.IDEE_GH_TOKEN }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+      - uses: ./.github/actions/gitConfig
+        with:
+          email: ${{ secrets.IDEE_GH_EMAIL }}
+      - name: Retrieve Scripts
+        run: |
+          git clone https://github.com/forcedotcom/bundle-publish-scripts.git
```

Comment:
Looks like this flow will be needed only as long as we are publishing the bundling changes as beta release. What happens after that? How can this flow be made more generic to release any changes as beta release

Reply:
publishBetaRelease.yml can still be manually triggered 😄